### ISSUE TITLE
AttributeError: module 'factory' has no attribute 'django'

### ISSUE THREAD

**Issue by ajbeach2** [FactoryBoy/factory_boy#769]
#### Description
I cannot import the DjangoModelFactory as described in the documentation. I am using factory_boy 3.0.0 and python 3.8.3:

```
->pip freeze | grep factory
factory-boy==3.0.0
 ->python --version
Python 3.8.3
```

#### To Reproduce
From the documentation, [https://factoryboy.readthedocs.io/en/3.0.0/orms.html](https://factoryboy.readthedocs.io/en/3.0.0/orms.html) I tried to create a class:
```
import factory
class UserFactory(factory.django.DjangoModelFactory):
    class Meta:
        model = 'myapp.User'  # Equivalent to ``model = myapp.models.User``
        django_get_or_create = ('username',)
```

I get the following error:

```
AttributeError: module 'factory' has no attribute 'django
```

**Comment by zymergen-stian** [comment]
See https://github.com/FactoryBoy/factory_boy/blob/master/docs/changelog.rst#300-2020-08-12, you need to rewrite your imports.

**Comment by zymergen-stian** [comment]
Also I want to point out that https://github.com/FactoryBoy/factory_boy/commit/7a2d57778f1d8167cbcfb2cf6d61558d7ee39abf does not work.

```py
try:
    import django
except ImportError:
    pass
```

This imports the "real django" pkg instead of the `django.py` of factoryboy. I believe the line should be `from . import django`?

**Comment by rbarrois** [comment]
@zymergen-stian OH, thanks for the report! I'll fix it right away!

**Comment by rbarrois** [comment]
Fixed in release 3.0.1.

**Comment by zymergen-stian** [comment]
Thank you so much for the quick fix!! Verified it works.
